### ISSUE TITLE
TypeError: driver.getAllRooms is not a function

### ISSUE THREAD

**Issue by xtagon** [screepers/screeps-server-mockup#22]
[screeps/driver](https://github.com/screeps/driver) version 5.1.0 has a breaking change that removes `driver.getAllRooms`, but this version is installed with screeps-server-mockup because the dependency allows it, which results in the following error:

```
TypeError: driver.getAllRooms is not a function
      at ScreepsServer.tick (node_modules\screeps-server-mockup\src\screepsServer.js:108:36)
```

The breaking commit is: https://github.com/screeps/driver/commit/18bcfce165b776d1b64ece229543b15dd3c97ddf

**Comment by xtagon** [comment]
Found a workaround: Pin the `screeps` package version to `4.0.5` in `package.json`.

**Comment by brisberg** [comment]
Hmm I'm going to bring this up over on the screeps/driver project, as that kind of breaking change shouldn't be included in a minor version update.

For the moment though, that's good that you found a work around. I'll see if I can update this project to work with the new driver API so you don't need to pin it.

**Comment by brisberg** [comment]
Thanks for pointing out this issue!

Got a PR up which fixes compatibility with screeps 4.1.5. Should be usable in the next release.

**Comment by xtagon** [comment]
Thank you for such a quick response!
